### ISSUE TITLE
Add more security-related actions and functionality 

### ISSUE THREAD

**Issue by frikky** [frikky/Shuffle-apps#169]
- [ ] CIDR contains IP
- [ ] Is item datatype X? (Use available observable types in TheHive/MISP)

**Comment by D4rkw0lv3s** [comment]
is item datatype like (string, integer, boolean, float)? Or like with regex to check if it is an mail, ip, domain, url, etc...?

**Comment by frikky** [comment]
> is item datatype like (string, integer, boolean, float)? Or like with regex to check if it is an mail, ip, domain, url, etc...?

Good idea. Datatype matches with autocasting should be able to handle the first part here easily
